sesam.py

```diff
@@ -916,8 +916,24 @@ def format_zip_config(self, zip_data, binary=False):
         buffer.seek(0)
         return buffer.read()
 
+
+    def set_authconfig_credentials(self, *args):
+        try:
+            auth_credentials = self.read_config_file(".authconfig")
+            for arg in args:
+                token=auth_credentials[arg]
+                if token.startswith('"') and token.endswith('"'):
+                    setattr(self.args, arg, token[1:-1])
+                else:
+                    setattr(self.args, arg, token)
+            logger.info("Found authentication credentials in .authconfig file.")
+        except KeyError:
+            logger.warning("Could not find %s in .authconfig file. Checking the arguments." % arg)
+
+
     def authenticate(self):
-        self.args.service_url, self.args.service_jwt = self.read_config_file(".syncconfig").values()
+        self.args.service_url=node_url
```

Comment:
I think this is a bit brittle, as you're relying on the scope of the calling method to have the `node_url` and `jwt_token` variables defined (which the __main__ method does on line 2414). But that method gets these variables from the method `get_node_and_jwt_token` which is a member method of this class, which sets these variables on self anyway (`self.jwt_token` and `self.node_url`). So you could just get it from `self`, it would be more robust.

Reply:
Done